@xen-orchestra/web-core/lib/components/table/ColumnTitle.vue

```diff
@@ -67,8 +67,8 @@ const { t } = useI18n()
 const router = useRouter()
 
 const interactions = computed<Interaction[]>(() => [
-  { id: 'sort-asc', icon: faArrowDown, label: t('core.sort.ascending'), disabled: true },
-  { id: 'sort-desc', icon: faArrowUp, label: t('core.sort.descending'), disabled: true },
+  { id: 'sort-asc', icon: faArrowDown, label: t('core.sort.ascending') },
+  { id: 'sort-desc', icon: faArrowUp, label: t('core.sort.descending') },
```

Comment:
Don't forget to remove the "coming soon" tooltip inside the template when the interaction is not _disabled_.